documentation/examples/prometheus.yml

```diff
@@ -27,4 +27,17 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ['localhost:9090']
+      - targets: 
+        - 'localhost:9090'
+
+  # An example for relabeling an instance
```

Comment:
how about making a lean version the default (which is then absolutely the first thing people come across) and have actual examples in the examples.yml?